packages/@jsii/dotnet-runtime/src/Amazon.JSII.Runtime/AssemblyInfo.cs

```diff
@@ -1,9 +1,8 @@
-using Castle.Core.Internal;
 using System;
 using System.Runtime.CompilerServices;
 
 [assembly:CLSCompliant(false)] // among other things, array parameters are passed to attributes
 
-[assembly:InternalsVisibleTo("Amazon.JSII.Runtime.IntegrationTests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100d7206fa09e081f23fa91938ceede7b25c0dfb8010fa55038a859a3e0150f652e655b8f0d9c17c41bb1d2b2c5ee2eb6af43a930cd8e80c2d7fe7703b24e66a1c674f6328e44bda8000b5df4686d77be2aeef092da587672a7ca9b9a9502ad960bcb7f35154d9a855bf9db546daf79a34c9fb5ecb23c6291a92587d8ece0354e9c")]
-[assembly:InternalsVisibleTo("Amazon.JSII.Runtime.UnitTests, PublicKey=0024000004800000940000000602000000240000525341310004000001000100d7206fa09e081f23fa91938ceede7b25c0dfb8010fa55038a859a3e0150f652e655b8f0d9c17c41bb1d2b2c5ee2eb6af43a930cd8e80c2d7fe7703b24e66a1c674f6328e44bda8000b5df4686d77be2aeef092da587672a7ca9b9a9502ad960bcb7f35154d9a855bf9db546daf79a34c9fb5ecb23c6291a92587d8ece0354e9c")]
-[assembly:InternalsVisibleTo(InternalsVisible.ToDynamicProxyGenAssembly2)]
+[assembly:InternalsVisibleTo("Amazon.JSII.Runtime.IntegrationTests")]
+[assembly:InternalsVisibleTo("Amazon.JSII.Runtime.UnitTests")]
+[assembly:InternalsVisibleTo("DynamicProxyGenAssembly2")]
```

Comment:
Is this intentionally different?

Reply:
Yes - you gotta refer to it "by the string" if you're not-strong-name-signed, and using the constant if you are. This is awkward, I agree.